tests/list.spec.ts

```diff
@@ -157,17 +157,35 @@ test('nested list blocks', async ({ page }) => {
   >
     <list
       prop:checked={false}
-      prop:text="123"
+      prop:text={
+        <>
+          <text
+            insert="123"
+          />
+        </>
+      }
       prop:type="bulleted"
     >
       <list
         prop:checked={false}
-        prop:text="456"
+        prop:text={
```

Comment:
Can we make it compatible with current `prop:text="456"` style? For now all the test cases have not involved in rich text.

Reply:
Ok, now it will print string if the text no any format